test/e2e/apimachinery/resource_quota.go

```diff
@@ -257,8 +285,14 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		err = waitForResourceQuota(f.ClientSet, f.Namespace.Name, quotaName, usedResources)
 		framework.ExpectNoError(err)
 	})
-
-	ginkgo.It("should create a ResourceQuota and capture the life of a configMap.", func() {
+	/*
+		Release: v1.16
+		Testname: ResourceQuota, object count quota, configmap
+		Description: Create a ResourceQuota. Creation MUST be successful and its ResourceQuotaStatus MUST match to expected used and total allowed resource quota count within namespace.
+		Create a ConfigMap. Its creation MUST be successful and resource usage count against the ConfigMap object MUST be captured in ResourceQuotaStatus of the ResourceQuota.
+		Delete the ConfigMap. Deletion MUST succeed and resource usage count against the ConfigMap object MUST be released from ResourceQuotaStatus of the ResourceQuota.
+	*/
+	framework.ConformanceIt("should create a ResourceQuota and capture the life of a configMap.", func() {
```

Comment:
Does this test really need to be so complicated, or did it just copy the secret test?

Reply:
Same question

Reply:
@mikedanese asked a similar question when this test was first introduced https://github.com/kubernetes/kubernetes/pull/68812/files#r230931734

Per comments below this is making an allowance for `ca.crt` to show up

Reply:
Why do we need to do that? Is this just related to how we have configured the e2e cluster? If there is this race condition in pod startup, it would seem that users can hit this too, we shouldn't just work around it.